android/CloudVision/app/src/main/java/com/google/sample/cloudvision/MainActivity.java

```diff
@@ -130,7 +153,10 @@ public void uploadImage(Uri uri) {
         if (uri != null) {
             try {
                 // scale the image to 800px to save on bandwidth
-                Bitmap bitmap = scaleBitmapDown(MediaStore.Images.Media.getBitmap(getContentResolver(), uri), 1200);
+                Bitmap bitmap =
+                        scaleBitmapDown(
+                                MediaStore.Images.Media.getBitmap(getContentResolver(), uri),
+                                1200);
```

Comment:
/shruggie I'm not sure. I think it was originally set to 800. I'm not sure in which PR it was updated.